ForgeUnity/Assets/BeardedManStudios/Scripts/Networking/Forge/Networking/Objects/NetworkObject.cs

```diff
@@ -954,7 +976,14 @@ public void InvokeRpc(NetworkingPlayer sender, ulong timestep, BMSByte data, Rec
 		/// <returns>If <c>true</c> the RPC will be replicated to other clients</returns>
 		protected virtual bool ServerAllowRpc(byte methodId, Receivers receivers, RpcArgs args)
 		{
-			return true;
+			if(IsServer && !AllowPublicRpcs) {
```

Comment:
Spacing after `if`, `{` on same line.
`else` statements not needed due to return statements above.

Reply:
yup I agree, that if structure can be simplified.
```suggestion
			if (IsServer && !AllowPublicRpcs)
			{
			    if (args.Info.SendingPlayer == Owner || args.Info.SendingPlayer == Master)
					return true;
					
			    return false;
			}
			
			return true;
```